splinepy/utils/data.py

```diff
@@ -651,6 +651,10 @@ def __setitem__(self, key, value):
         elif "PyMultipatch" in str(type(value).__mro__):
             self._saved[key] = len(self._helpee.fields)
             self._helpee.add_fields([value])
+        elif self._helpee.name.startswith("Multipatch"):
+            # also accept integers as a reference to the field
+            if isinstance(value, int):
+                self._saved[key] = value
```

Comment:
```suggestion
        elif isinstance(value, int):
            # also accept integers as a reference to the field
            self._saved[key] = value
```
Well, this is a member func of multipatch data, meaning, helpee should always be multipatch.

Reply:
Yes, thank you for pointing that out